osu.Game/Screens/OnlinePlay/Multiplayer/Match/ConfirmAbortDialog.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using osu.Framework.Graphics.Sprites;
+using osu.Game.Overlays.Dialog;
+
+namespace osu.Game.Screens.OnlinePlay.Multiplayer.Match
+{
+    public partial class ConfirmAbortDialog : PopupDialog
```

Comment:
Could possibly inherit `DangerousActionDialog` instead and just change the `HeaderText` for more consistent copy.

Reply:
Have done this, it shared localisations and reduces implementation overhead.